### DISCUSSION TITLE
Why use Supabase Auth over the RealtimeDB for user authentication?

### DISCUSSION THREAD

**Discussion by Kleysley** [supabase/supabase#1712] [Questions] [answered]
What is the advantage of using Supabase Auth over just creating a "users" table in the realtimeDB and using SQL to log people in or grant them acces to certain data?
What would Supabase Auth give me that a users-and-hashed-passwords-table wouldn't?

**Comment by inian** [comment, marked as answer]
Hey @niklase2, few advantages of using Supabase Auth off the top of my head
- Integration with a (growing) list of third party vendors like Twitter, Github, Facebook, MagicLinks etc
- Client libraries with [simple APIs](https://supabase.io/docs/reference/javascript/auth-signup) to interact with your backend
- Our Auth integrates with other Supabase services like Storage 
- Common workflows like password reset, change password is built in support for sending support for sending emails
- Audit trails for all Auth related actions

Auth is one of those things that seems deceptively simple at first sight but can be difficult to get right. However, you can roll out our own Authentication too, especially if you aren't interested in OAuth, etc since you have full access to the Postgres Database.
